apps/mobile-wallet/src/components/NFTsGrid.tsx

```diff
@@ -53,16 +55,18 @@ const NFTsGrid = ({ addressHash, nfts: nftsProp, nftSize, nftsPerRow = 3 }: NFTs
 
   return (
     <FlashList
+      {...props}
       data={data}
       keyExtractor={(item) => item.id}
       renderItem={({ item: nft }) => (
         <NFTThumbnailContainer key={nft.id}>
           <NFTThumbnail nftId={nft.id} size={size} />
         </NFTThumbnailContainer>
       )}
-      contentContainerStyle={{ paddingHorizontal: containerHorizontalPadding }}
+      contentContainerStyle={{ paddingHorizontal: containerHorizontalPadding, paddingBottom: 100 }}
       numColumns={columns}
       estimatedItemSize={64}
+      scrollEnabled={scrollEnabled}
```

Comment:
this can be deleted since it can be part of the `props` deconstruction.

Reply:
Was this removed in the next PRs?

Reply:
Noted, I'll remove it. IMO it's a nit and shouldn't block a PR.